CoreFindView/test/unit/src/au/gov/asd/tac/constellation/views/find2/FindViewControllerNGTest.java

```diff
@@ -258,50 +260,13 @@ public void testRetriveMatchingElements() {
                 when(pluginExecution.executeLater(Mockito.any(Graph.class))).thenReturn(null);
                 mockedStaticPlugin.when(() -> PluginExecution.withPlugin(Mockito.any(Plugin.class))).thenReturn(pluginExecution);
 
-                instance.retriveMatchingElements(true, false);
+                basicFindPlugin = new BasicFindPlugin(instance.getCurrentBasicFindParameters(), true, false);
+                PluginExecution.withPlugin(basicFindPlugin).executeLater(graph);
                 verify(pluginExecution, times(2)).executeLater(Mockito.eq(graph));
-                verify(pluginExecution).executeLater(Mockito.eq(graph2));
-
             }
         }
     }
 
-//            // Populate the attribute list with the attributes of the given type
```

Comment:
just curious as to the reason for this whole code block being commented out?

Reply:
Not really sure why, it was like that before I looked at it

Reply:
ah right, I just realised you were removing that code. All good